### PR TITLE
JA: add site/ja/tutorials/load_data/unicode.ipynb

### PR DESCRIPTION
**Pull request by yyone**
original document:
https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/unicode.ipynb

### CODE REVIEW

**Comment by masa-ita** [comment]
@yyone 
Great work!
I'd like to leave some comments.  Please check them.

**Comment by masa-ita** [comment]
It seems that the file site/en/tutorials/load_data/unicode.ipynb should not be included in this PR.

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 1]
```diff
@@ -0,0 +1,813 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AXH1bmUctMld"
+   },
+   "source": [
+    "# Unicode 設定\n",
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Suggestion:
The title "Unicode strings" should be "Unicode 文字列".

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 2]
```diff
@@ -0,0 +1,813 @@
...
+   "metadata": {},
+   "source": [
+    "Note: これらのドキュメントは私たちTensorFlowコミュニティが翻訳したものです。コミュニティによる 翻訳は**ベストエフォート**であるため、この翻訳が正確であることや[英語の公式ドキュメント](https://www.tensorflow.org/?hl=en)の 最新の状態を反映したものであることを保証することはできません。 この翻訳の品質を向上させるためのご意見をお持ちの方は、GitHubリポジトリ[tensorflow/docs](https://github.com/tensorflow/docs)にプルリクエストをお送りください。 コミュニティによる翻訳やレビューに参加していただける方は、 [docs-ja@tensorflow.org メーリングリスト](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ja)にご連絡ください。"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 紹介\n",
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Weak suggestion:
In this context the word "introduction" means like "序".  So how about "はじめに".

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 3]
```diff
@@ -0,0 +1,813 @@
...
+    "\n",
+    "自然言語モデルは、しばしば異なる文字セットを使った異なる言語を扱います。 *Unicode*は、ほぼすべての言語で文字表示に使われている標準的なエンコードの仕組みです。各文字は、`0` から`0x10FFFF`までの一意の整数の [コードポイント（符号位置）](https://ja.wikipedia.org/wiki/%E7%AC%A6%E5%8F%B7%E7%82%B9) を使ってエンコードされます。1つの *Unicode文字*は、ゼロ以上のコードポイントを持った文字列になります。\n",
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Suggestion:
"1つの *Unicode文字*は、ゼロ以上のコードポイントを持った文字列になります。" -> "1つの*Unicode 文字列*は、ゼロ個以上のコードポイントのシーケンスです。"

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 4]
```diff
@@ -0,0 +1,813 @@
...
+    "\n",
+    "このチュートリアルでは、どのようにTensorFlowでUnicode文字列を表現し、どのように標準的な文字列操作と同じように、それらのUnicode文字列を操作するか、その方法を示します。そして、スクリプト種別に基づいて、Unicode文字列をトークンに分解します。"
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Week suggestion:
How about this,
"このチュートリアルでは、TensorFlow での Unicode 文字列の表現方法と、どうやって Unicode で標準的な文字列操作と同様の操作を行うかについて示します。また、スクリプト検出にもとづいて Unicode 文字列をトークンに分解します。"

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 5]
```diff
@@ -0,0 +1,813 @@
...
+   "outputs": [],
+   "source": [
+    "tf.constant(u\"Thanks 😊\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "バイト列は atomic 単位で扱われるため、`tf.string` 型のテンソルは可変長のバイト文字列を扱います。また、文字列長はテンソルの次元には含まれません。"
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Week suggestion:
"バイト列は atomic 単位で扱われるため、`tf.string` 型のテンソルは可変長のバイト文字列を扱います。" -> "バイト列が最小限の単位として扱われるため、`tf.string`  型のテンソルは可変長のバイト文字列を保持できます。"

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 6]
```diff
@@ -0,0 +1,813 @@
...
+    "                             output_encoding='UTF-16-BE')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### バッチの次元\n",
+    "\n",
+    "複数の文字列をデコードする場合、各文字列の文字数が等しくない場合があります。返される結果は[`tf.RaggedTensor`](../../guide/ragged_tensor.ipynb)であり、最も深い次元の長さは各文字列の文字数によって異なります。:"
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Weak suggestion:
"最も深い" -> "最も内側の"

File: site/ja/tutorials/load_data/unicode.ipynb [hunk 7]
```diff
@@ -0,0 +1,813 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "mx7HEFpBzEsB"
+   },
+   "source": [
+    "## 例：シンプルなセグメンテーション\n",
+    "\n",
+    "セグメンテーションは、テキストを単語のような粒度に分割するタスクです。これは、スペース文字を使用して単語を区切れる場合には簡単に行えますが、一部の言語（中国語や日本語など）はスペースを使いませんし、また、一部の言語（ドイツ語など）には、意味を区別するために分ける必要がある、単語を結合した長い単語（テキスト）があります。Webテキストでは、「NY株価」（ニューヨーク株価）のように、異なる言語とスクリプトがしばしば混在しています。\n",
```
Comment (masa-ita) [inline site/ja/tutorials/load_data/unicode.ipynb]: Weak suggestion:
"意味を区別する" -> "意味を解析する"
"長い単語（テキスト）" -> "長い複合語"

**Comment by yyone** [comment]
@masa-ita
Thank you for your reply!!
I fixed your suggestion. Please check it again.

but, I can't understand this below:
'''
It seems that the file site/en/tutorials/load_data/unicode.ipynb should not be included in this PR.
'''
does it need english page file in this PR ??

**Comment by masa-ita** [comment]
@yyone 
I've checked the changes you made. 

About the file site/en/tutorials/load_data/unicode.ipynb, it seems that you open the file with jupyter during translation and it was saved accidentally.
I think you'd better revert this file to prevent unneeded conflict.

**Comment by kiszk** [comment]
I also think that a PR for Japanese translation does not need to include the original file in English (i.e. https://github.com/tensorflow/docs/pull/1312/files#diff-2f88a805d030e30d0a9d773a955226e3).

**Comment by yyone** [comment]
@masa-ita, @kiszk 
Thank you for your check.
I reverted "en/.../unicode.ipynb" that i have mistaken to commit.

**Comment by masa-ita** [comment]
@yyone 
It seems OK.  LGTM!

**Pull request merged**